platform/packages/platform/src/response.rs

```diff
@@ -8,6 +8,14 @@ pub fn empty_response() -> CwResponse {
     response_only_messages(MessageResponse::default())
 }
 
+pub fn response_consuming<T, E>(response: T) -> Result<CwResponse, E>
```

Comment:
why do we need another variant of `fn response`? How have you decided which one to use? It looks like each of them alone is enough.

Reply:
`origin/main` was broken because `response` expected a reference but `ReleaseLabel` was not behind one. The motivation behind this was to avoid having to write it as a closure just to make it a reference everywhere.

Reply:
disagree it was "broken"
two versions of `fn response` are too much, please choose and stick to one.

Reply:
I don't know what do you disagree with. Please check the signatures.

I wanted to avoid repetition. If you think using lambdas/closures would be better, I will change it.